### ISSUE TITLE
build scans command is wrong

### ISSUE THREAD

**Issue by bodhili** [gradle/gradle#28480]
### Issue type

Wrong or misleading information

### Problem description

Gradle version : 8.6

FAQ: the title of "Core Concepts"  and the tenth title named "Build scans" that content `./gradlew build --scans` is wrong,  and I think of the content ` ./gradlew build --scan` is right

Link: [https://docs.gradle.org/current/userguide/build_scans.html#enable_build_scans](url)

### Page with the problem

https://docs.gradle.org/current/userguide/build_scans.html#enable_build_scans

**Comment by RajeevDash007** [comment]
![build_scans-adoc-gradle-Codespaces-RajeevDash007-gradle-space-Visual-Studio-Code](https://github.com/gradle/gradle/assets/96860040/5e9cd800-4b3e-432d-8cc1-df8e67a62f67)
but its changed in the codebase, do you know why it isnt reflecting in the frontend?
@bodhili

**Comment by bodhili** [comment]
> ![build_scans-adoc-gradle-Codespaces-RajeevDash007-gradle-space-Visual-Studio-Code](https://private-user-images.githubusercontent.com/96860040/313442000-5e9cd800-4b3e-432d-8cc1-df8e67a62f67.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTA2NTY4MjMsIm5iZiI6MTcxMDY1NjUyMywicGF0aCI6Ii85Njg2MDA0MC8zMTM0NDIwMDAtNWU5Y2Q4MDAtNGIzZS00MzJkLThjYzEtZGY4ZTY3YTYyZjY3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzE3VDA2MjIwM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTMyYzdhMTIwZGExMGE1MjE5ZDNkNzA4YTc5ZGNhYjFlNmMzNDg0MGRiM2UyNWUyNmM3YmNlZjQxNDkxMjQ3YjMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.DECXvTfetpZCaQhPVCJDQAKWKZagfdPae1znF3Di8e0) but its changed in the codebase, do you know why it isnt reflecting in the frontend? @bodhili

Its maybe would update at the documentation version of gradle@8.7 or later.

**Comment by ov7a** [comment]
Indeed, it will be fixed with 8.7 update. The change is both in the master (next release, 8.8) and release branches (current release, 8.7, it's in RC-phase). You can see the updated docs in the [RC docs](https://docs.gradle.org/8.7-rc-3/userguide/build_scans.html#enable_build_scans).

Closing as a duplicate of
* https://github.com/gradle/gradle/issues/28342
